test/activation.jl

```diff
@@ -101,7 +113,7 @@ end
     end
 
     @testset "Test Integer64 and Integer32 inputs will force Float64 outputs" begin
-        test_value_int_input_forces_float64.(filter(x -> (x != relu && x != relu6), ACTIVATION_FUNCTIONS))
+        test_value_int_input_forces_float64.(filter(x -> (x != relu && x != relu6 ), ACTIVATION_FUNCTIONS))
```

Comment:
I think this extra space is a mistake.

Reply:
Oh! Yes, That was left there by mistake.

Reply:
I will remove this